.credo.exs

```diff
@@ -166,6 +166,7 @@
            excluded: [
              "lib/elixir_internal_certificate/account/accounts.ex",
              "lib/elixir_internal_certificate/scraper/scrapers.ex",
+             "lib/elixir_internal_certificate_web/controllers/api/v1/user_session_controller.ex",
```

Comment:
Might I know why we need to skip this file 🤔

Reply:
Because the credo checking on muti-alias, it happens like the very first PRs